Dockerfile

```diff
@@ -5,6 +5,8 @@ ENV GOOS=linux
 ENV GOARCH=amd64
 ENV CGO_ENABLED=0
 
+ARG TAG="dev"
```

Comment:
Can you please add a comment here that the value of `TAG` changes in `./bin/build`? it's not very clear without it.